app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,209 @@
+
+    <string name="toast_error_loading_image">Resim yüklenirken hata ile karşılaşıldı</string>
+    <string name="toast_no_files">Gösterilecek bir dosya bulunmamaktadır</string>
+
+    <string name="toast_paused">Durduruldu</string>
+    <string name="toast_resumed">Yeniden başlatıldı</string>
+    <string name="toast_completed">Tamamlandı</string>
+
+    <string name="toast_input_blocked">Kontroller kilitlendi. Basılı tutarak açın</string>
+    <string name="toast_input_allowed">Kontroller açıldı</string>
+
+    <string name="long_loading_warning">%1$s yüklemesi uzun zaman alıyor.</string>
+    <string name="long_loading_skipping">%1$s yüklemesi uzun zaman alıyor. Atlanıyor.</string>
+    <string name="long_loading_skip_action">Atla</string>
+
+    <!-- Controls -->
+    <string name="controls_title">&appname; Kontroller</string>
+    <string name="controls_content1"> Slayt gösterisi sırasında kullanılabilecek gizli kontroller
+        bulunmaktadır</string>
+    <string name="controls_content2">Sonraki / önceki görsele geçmek için *Sola / sağa kaydır*.
+    </string>
+    <string name="controls_content3">Slayt gösterisini başlatmak / durdurmak için *Bir kere dokun*.
+    </string>
+    <string name="controls_content5">Ayrıntı panelini göstermeden slayt gösterisini durdurmak içn
+        *Çift dokun*.</string>
+    <string name="controls_content4">Kullanıcı kontrolünü devre dışı bırakmak için *Slayt gösterisi
+        sırasında uzunca dokun*. Babaannenizin cihazın tozunu aldığı sırada yanlışlıkla slayt
+        gösterisini durdurmaması için kullanışlı bir seçenek.</string>
+
+    <!-- Image descriptions -->
+    <string name="image_description">Resim</string>
+    <string name="logo_description">Logo</string>
+
+    <!-- Credits -->
+    <string name="title_activity_credits">Hakkında &appname;</string>
+    <string name="credits_creator"><a href="https://michael.standen.link">&author;</a> tarafından
+    </string>
+    <string name="credits_content1">&appname; Telefonunuzdaki resimleri görüntülemek için kolay ve
+        kullanışlı bir uygulama </string>
+    <string name="credits_content2">Gereksiz karmaşıklıktan uzak bu uygulama piyasadaki en kolay ve
+        kullanışlı slayt gösterisi uygulamasıdır.
+	</string>
+    <string name="credits_content3">Buradan kaynak kodunu görebilir, hata bildirebilir ya da yeni
+        özellik talep edebilirsiniz:
+		<a href="https://github.com/Screaminghawk/android-slideshow">Github</a>.
+	</string>
+    <string name="credits_content4">Bu uygulama tamamen ücretsiz olup, açık kaynaklıdır;
+        <a href="https://tldrlegal.com/license/mit-license">MIT lisansı ile lisanslanmıştır.</a>.
+	</string>
+    <string name="credits_content5">Teşekkürler <a href="https://github.com/psyanite">Yan Tsui</a>
+		 &appname; ikonunun test ve tasarımı için.
+	</string>
+    <string name="credits_content6">&appname; çevirisi
+		<a href="https://michael.standen.link">&author;</a> tarafından size ulaştırılmıştır.
```

Comment:
Thanks for the PR! This line is here for you to add your details so that you will be credited in the app. Please update with your name and website if you want credits :) Otherwise you can remove this string

Reply:
Hi, sorry for the late edit :) I removed the part with credits.